content/en/docs/contributing/localization.md

```diff
@@ -18,22 +18,44 @@ guidance offered in this section.
 
 ### Summary / TL;DR {#summary}
 
-Do translate:
+#### ✅ Do {#do}
```

Comment:
Suggestion: How about a table to present the list? I fear some might jump straight to the the Do NOT and think it's DOs.

Reply:
> I fear some might jump straight to the the Do NOT and think it's DOs.

I'm willing to give readers the responsibility of not making such an obvious mistake.

That being said, that's why I've included an icon on the heading and the differing background color for the section content. I'd prefer not putting it in a table, b/c it is harder to get responsive table content, etc.